cmd/client/cmd/service.go

```diff
@@ -2,9 +2,11 @@ package cmd
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"log"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/jedib0t/go-pretty/v6/table"
```

Comment:
Let's use this one instead https://github.com/rodaine/table . It's simpler and more compact output.